e2e/mobile/specs/verifyAddress/receiveFlow.spec.ts

```diff
@@ -1,60 +1,91 @@
+import { ReceiveFundsOptions } from "@ledgerhq/live-common/lib/e2e/enum/ReceiveFundsOptions";
+
 describe("Receive Flow", () => {
   const account = Account.ETH_1;
 
   beforeAll(async () => {
     await app.init({
       speculosApp: account.currency.speculosApp,
       userdata: "EthAccountXrpAccountReadOnlyFalse",
+      featureFlags: {
+        llmModularDrawer: {
+          enabled: true,
+          params: {
+            receive_flow: true,
+          },
+        },
+        noah: {
+          enabled: true,
+        },
+      },
     });
 
     await app.portfolio.waitForPortfolioPageToLoad();
   });
 
-  async function openReceive() {
+  beforeEach(async () => {
     await app.portfolio.openViaDeeplink();
     await app.portfolio.waitForPortfolioPageToLoad();
-    await app.receive.openViaDeeplink();
-  }
+    await app.portfolio.tapQuickActionReceiveButton();
+    await app.receive.selectReceiveFundsOption(ReceiveFundsOptions.CRYPTO);
+  });
+
+  afterEach(async () => {
+    await app.modularDrawer.tapDrawerCloseButton({ onlyIfVisible: true });
+  });
 
   $TmsLink("B2CQA-1858");
   $TmsLink("B2CQA-1860");
   it("Should display the number of account existing per networks", async () => {
-    await openReceive();
-    await app.receive.selectAsset("ETH");
-    await app.receive.expectNumberOfAccountInListIsDisplayed("ethereum", 3);
-    await app.receive.expectNumberOfAccountInListIsDisplayed("optimism", 1);
```

Comment:
I think `expectNumberOfAccountInListIsDisplayed` is no more used and could be removed